components/Image/index.jsx

```diff
@@ -106,7 +108,7 @@ Image.defaultProps = {
   height: 'auto',
   src: '',
   title: '',
-  width: 'auto',
+  width: 'smart',
```

Comment:
thanks for the suggestion @kellyjosephprice This should map to [this line](https://github.com/readmeio/markdown/blob/78e5fd51202fb8b7a0cce527a84ef2313a7ccdf2/components/Image/style.scss#L43)